server/src/instant/db/model/attr.clj

```diff
@@ -532,40 +534,39 @@
      conn
      (hsql/format
       {:with (concat
-              (if-let [attr-table-updates
-                       (seq (changes-that-require-attr-model-updates updates))]
```

Comment:
Why don't we need to run this check any more?

Reply:
Because all changes require attr-model-updates now. It was used before to separate changes that only touched `idents`, but now if you touch `idents` you touch attrs too